CrazyCanvas/Source/States/PlaySessionState.cpp

```diff
@@ -334,6 +228,23 @@ bool PlaySessionState::OnPacketReceived(const LambdaEngine::PacketReceivedEvent&
 
 		m_pLevel->CreateObject(ESpecialObjectType::SPECIAL_OBJECT_TYPE_PLAYER, &createPlayerDesc);
 
+#if 1
+		// Create a player to shoot at
+		robotAnimationComp.Pose.pSkeleton = ResourceManager::GetMesh(robotGUID)->pSkeleton;
+		if (animationsExist)
+		{
+			robotAnimationComp.pGraph = DBG_NEW AnimationGraph(DBG_NEW AnimationState("walking", animations[0]));
+		}
+
+		createPlayerDesc.IsLocal	= false;
+		createPlayerDesc.TeamIndex	= 2;
+		createPlayerDesc.Position.x = -3.0f;
+		createPlayerDesc.Position.y = 0.75f;
+		createPlayerDesc.Position.z = -3.0f;
+		createPlayerDesc.NetworkUID += (int32)1;
+		createPlayerDesc.AnimationComponent = robotAnimationComp;
+		m_pLevel->CreateObject(ESpecialObjectType::SPECIAL_OBJECT_TYPE_PLAYER, &createPlayerDesc);
+#endif
```

Comment:
This is perfectly suited for the sandbox state.

Reply:
True